lib/node_modules/@stdlib/math/base/special/cospi/src/main.c

```diff
@@ -0,0 +1,84 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+/*
+* Notes:
+*	=> cos(-x) = cos(x)
+*	=> sin(-x) = -sin(x)
+*	=> cos(π/2) = 0
+*	=> cos(0) = 1
+*	=> cos(π) = -1
+*/
```

Comment:
I'll change this based on our conclusion at https://github.com/stdlib-js/stdlib/pull/2610#discussion_r1679467938